themes/default/style-guide.md

```diff
@@ -0,0 +1,43 @@
+# Style Guide
+This document defines some general styles we adhere to in the docs.
+
+## Words are important
+
+- Instead of _guys_ try _folks_, _yall_, or _everyone_
+- Instead of _crazy_ try _wild_
+- Instead of _click_ use _select_
+- Instead of _dummy_ use _placeholder_
+- Avoid using gendered language
+- Avoid using violent language (e.g., _kill_)
+- Avoid pop-culture references
+
+## Links
+
+- Link text should be descriptive and be understandable outside of the context that it is in.
+- Avoid link text like _here_, _click here_, _see here_
+
+## Notes and Warning
+
+- Use notes to communicate important information
+- Try and limit the use of notes within a single page
```

Comment:
@cnunciato i mean on a single page. ideally if you have 3 paragraphs and 10 notes that can be pretty difficult to read.

Reply:
Yep, ok, makes sense. So would this work? (Otherwise, it reads a bit like "don't use notes", where really we mean, "don't use several notes".)

```suggestion
- Try and limit the number of notes within a single page
```